### PR TITLE
Feature/mobile menu

### PR DESCRIPTION
**Pull request by ahmetbozaci**
I add simple function for open menu in mobile version.

### CODE REVIEW

**Comment by george-swift** [comment]
## Invalid Pull Request ❌ 

Hello Ahmet,

Before this project can be reviewed, kindly ensure that there are no linter errors. I'm sure you must have put in a lot of work but passing linters is a general requirement before any review. After fixing them, you can go ahead and ask for a code review.
You should get a review following a valid request in less than 12 hrs. Godspeed!



**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-swift — CHANGES_REQUESTED** [review]
## Invalid Pull Request ❌ 

Hello Ahmet,

Before this project can be reviewed, kindly ensure that there are no linter errors. I'm sure you must have put in a lot of work but passing linters is a general requirement before any review. After fixing them, you can go ahead and ask for a code review.
You should get a review following a valid request in less than 12 hrs. Godspeed!




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -6,17 +6,18 @@
...
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <h1 id="logo">Ahmet Bozacı</h1>
+      <button type="button" class="menu-bars" onclick="openMenu()" >
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
```
Comment (Berabjesus) [inline index.html:18]: - [ ] When a user click (or tap) the hamburger button, the mobile menu should appear. Currently, it is just moving the hamburger button to the left.
image

- Kindy work on this and it's best if you check other functionalities are met. For example: - When a user clicks on the buttons inside the menu, the menu should disappear and also make sure the styling matches the one specified on [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).

**Review by Berabjesus — CHANGES_REQUESTED** [review]
# Status: CHANGES REQUIRED :warning: :construction: 
Hey @ahmetbozaci great job working on the previous review 👏 👏 
### You have a few changes to carry out before we proceed so please take a look at the things you have to work on and their respective review comments below.

## Required Changes ♻️

- [ ] When a user click (or tap) the hamburger button, the mobile menu should appear. (check inline comment below for further information).

----------
#### **Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**
----------
#### As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
----------
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

File: script.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
+function openMenu() {
+  const menu = document.getElementById('desktop-nav');
+  if (menu.style.display === 'none') {
+    menu.style.display = 'block';
+  } else {
+    menu.style.display = 'none';
+  }
+}
```
Comment (ccobasi) [inline script.js:9]: - [ ] I see you are trying to implement openMenu function. But it seems something is not right. When I clicked the hamburger menu it does not display the mobile menu. I suggest you see [DOM manipulation](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Manipulating_documents) documentation. That will help you to implement the necessary requirements.
- [ ] I also suggest you read the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md).

**Review by ccobasi — COMMENTED** [review]
Hi [@ahmetbozaci](https://github.com/ahmetbozaci), 

Good job so far! You have a good mastery of git-flow.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  It is required that you make your readme file look professional. See Microverse [readme template](https://github.com/microverseinc/readme-template).

_Check the comments under the review._

## Optional suggestions
[OPTIONAL] 

- In desktop, when the user scrolls you need to check if the user scrolled to a new section. If so, you should highlight the name of the section in the menu by manipulating the DOM using CSS classes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**